README.md

```diff
@@ -246,6 +250,7 @@ Yazılım geliştirme ile ilgili özenle seçilmiş Türkçe kaynaklar listesi.
 - [Arın Yazılım](https://www.youtube.com/channel/UCzVB3qcyTd-QGJKY_Hpky5g/)
 - [Ahmet Yasin Ak](https://www.youtube.com/channel/UC06Q0ZtwUCd2uY4-2KUpUIg/feed)
 - [Data İstanbul](https://www.youtube.com/channel/UCmIscBfFWiT1avWgSCwJIeg)
+- [Ömer Faruk Çolakoğlu](https://www.youtube.com/user/ocolakoglu)
```

Comment:
Bu linkte kişinin beğendiği alakasız videolar da çıkabiliyor, bu yüzden daha önce de belirttiğim gibi ( #77 ) /videos URL'i olsa daha uygun olur diye düşünüyorum: https://www.youtube.com/user/ocolakoglu/videos